### ISSUE TITLE
Where is the solution of Section 2  exercise????? 

### ISSUE THREAD

**Issue by DimitraK92** [mosh-hamedani/vidly-mvc-5#208]
The GitHub account seems so messed up to me. The solutions of the exercises should have been inside the course online or attached to each video exercise. I've wasted so much time on searching here for the solution and i still haven't found it.

**Comment by brunaselymes** [comment]
https://github.com/mosh-hamedani/vidly-mvc-5/commit/b727a26e1b4b88abe84a8b48208fec537db2ed43

**Comment by DimitraK92** [comment]
Thank you very much. Till now, I had figured it out, though. But thanks anyway.
The issue is that I face problems all along the course. Errors are constantly coming up even though I've written the same code as the tutor or small differences which are not false. The versions of scripts and CSS files are also the same. I don't understand why this is happening. Nevertheless, I noticed that as the course moves on Mosh has made changes without telling them during the lesson, which makes it difficult to keep up some times.
